guides/advanced-use/debugging.md

```diff
@@ -1,6 +1,6 @@
 <!-- alex disable just -->
-For most Ember applications, Ember CLI "just works". Run `ember server` in your Terminal and you get
-a LiveReload development server at `http://localhost:4200`. Run `ember build`, and you get a `dist/`
+For most Ember applications, Ember CLI "just works". Run `ember server` or `npm start` in your Terminal and you get
```

Comment:
Can we try to make it more generalised like use package manager whatever your project is using instead of just `npm start`

Reply:
We're going to merge this unchanged, our reasoning is that if you are using any other package manager you are likely going to know how to "translate" commands from `npm` to your chosen package manager 👍